bioio_imageio/tests/test_reader.py

```diff
@@ -15,10 +15,10 @@
 @pytest.mark.parametrize(
     "filename, set_scene, expected_shape, expected_dims_order",
     [
-        ("example.bmp", "Image:0", (480, 640, 4), "YXS"),
+        ("example.bmp", "Image:0", (480, 640, 3), "YXS"),
         ("example.png", "Image:0", (800, 537, 4), "YXS"),
         ("example.jpg", "Image:0", (452, 400, 3), "YXS"),
-        ("example.gif", "Image:0", (72, 268, 268, 4), "TYXS"),
```

Comment:
why'd we lose a scene here?

Reply:
S is not scene, it's "Samples", as in 3 means RGB and 4 means RGBA